pico_limbo/src/configuration/tablist.rs

```diff
@@ -0,0 +1,12 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Default)]
+#[serde(default)]
+#[serde(deny_unknown_fields)]
+pub struct TabListConfig {
+    /// Header of the tab list displayed when the player presses the tab key.
+    pub header: String,
+
+    /// Footer of the tab list displayed when the player presses the tab key.
+    pub footer: String,
```

Comment:
Would be cool if you define a custom implementation for the default with the values you suggested:

```rust
impl Default for TabListConfig {
  fn default() -> Self {
    Self {
      header: "<bold>Welcome to PicoLimbo</bold>".to_string(),
      footer: "<green>Enjoy your stay!</green>".to_string(),
    }
  }
}
```